bedrock/base/templates/base-error.html

```diff
@@ -11,9 +11,10 @@
 
 {% block site_header %}
   <header class="mzp-l-content header-spacing">
-    <div class="header-image">
-    {{ high_res_img('img/co-brand/firefox_moz_co-brand_sm.png', {'alt': 'Firefox, Mozilla'}) }}
+    <div class="header-image" style="width: 145px;">
```

Comment:
It's better to avoid inline `style` attributes and instead do this styling in the CSS file.